syntax = "proto3";

message Article {
    string title = 1;
    string link = 2;
    string id = 3;
    string publication_date = 4;
    string description = 5;
    string source = 6;
    string search_keyword = 7;
    string search_url = 8;
}